sms_toolkit/messages/profiling.py

```diff
@@ -14,12 +14,21 @@ def profile_message(message):
 
         Args:
             message (basestring): The raw message string to profile.
+            is_for_mms (bool): Is this message for mms sending.
         Returns:
             (dict) A dict with the profiling results.
     """
     encoding = utils.determine_encoding_for_string(message)
 
-    if encoding == constants.GSM_ENCODING:
+    if is_for_mms:
+        segments = split_message_into_segments(
```

Comment:
Confused a bit here. Correct me if I'm wrong but I thought we were only sending one MMS. Once they hit 1600 characters, that's it. No segments beyond that. Or do segments in the `sms_toolkit` refer to something else?

Reply:
This will only make one segment, always, of 1600 or less. It preserves the format we're using now, and it makes sure to follow the expected limits of MMS.